src/types/DataTypes.sol

```diff
@@ -54,7 +54,7 @@ library Batch {
         uint128 totalAmount;
         bool cancelable;
         bool transferable;
-        LockupLinear.Range range;
         Broker broker;
+        LockupLinear.Range range;
```

Comment:
Why'd you move this below `broker`? Doesn't it break consistency?

Reply:
Oh, this is a mistake. Good catch.